### PR TITLE
fix #74006 supersoluce.com

### LINKED ISSUE #74006: supersoluce.com
### Issue URL (Anti Adblock Script)
 [https://www.supersoluce.com/soluce/assassin-s-creed-valhalla/soluce-assassin-s-creed-valhalla](https://adguardteam.github.io/AnonymousRedirect/redirect.html?url=https%3A%2F%2Fwww.supersoluce.com%2Fsoluce%2Fassassin-s-creed-valhalla%2Fsoluce-assassin-s-creed-valhalla)

### Screenshots
<details>
  <summary>Screenshot 1</summary>

  Screenshot 1
</details>

### System configuration

Information | value
--- | ---
AdGuard product: | AdGuard for Safari v        1.8.10
Browser: | Safari
Filters: | <b>Ad Blocking:</b><br/> AdGuard Base<br/><br/><b>Privacy:</b><br/> AdGuard Tracking Protection<br/><br/><b>Annoyances:</b><br/> AdGuard Annoyances<br/><br/><b>Language-specific:</b><br/> AdGuard French
Other extensions used: | Suite de sécurité Orange

### PR DESCRIPTION
**Pull request by wout-codes**
#74006

### CODE REVIEW

File: AnnoyancesFilter/sections/antiadblock.txt [hunk 1]
```diff
@@ -382,13 +382,13 @@ usrowing.org,stevensducks.com##.sidearm-adblock
...
...m,blic.rs,blitz.bg,bloomberght.com,bnd.com,brantfordexpositor.ca,business.financialpost.com,canoe.com,cinemablend.com,cnnturk.com,contra.gr,digitaltrends.com,dumpert.nl,dziennikbaltycki.pl,elmundo.es,elnuevoherald.com,esquire.com.gr,expressilustrowany.pl,fredericksburg.com,fresnobee.com,haberturk.com,hnonline.sk,ign.com,index.hr,interia.pl,izismile.com,jetphotos.com,kansascity.com,kentucky.com,klix.ba,kupujemprodajem.com,ladylike.gr,latimes.com,lecturas.com,lifestyle.bg,listenonrepeat.com,maisfutebol.iol.pt,marca.com,marketscreener.com,metropoles.com,money.bg,montrealgazette.com,mtlblog.com,mynet.com,narcity.com,nationalpost.com,news.bg,news247.gr,ni.dk,ntv.com.tr,nwitimes.com,observador.pt,okdiario.com,omaha.com,phonearena.com,polskatimes.pl,poststar.com,sanluisobispo.com,showtv.com.tr,slickdeals.net,sme.sk,snopes.com,sport24.gr,star-telegram.com,startv.com.tr,sunherald.com,tdn.com,theindependent.com,theobserver.ca,theprovince.com,thesouthern.com,thestate.com,tomshw.it,topito.com,topsport.bg,torontosun.com,trib.com,uol.com.br,vancouversun.com,wcfcourier.com,windsorstar.com,ynet.co.il,yourdictionary.com,zonebourse.com#%#//scriptlet("abort-current-inline-script", "atob", "googlefc")
```
Comment (zloyden) [inline AnnoyancesFilter/sections/antiadblock.txt:391]: Did you check it? It doesn't work on my end, so remove the domain.
Also please add `@@||supersoluce.com^$generichide` a bit lower to the
```
!#if (adguard_app_ios || adguard_ext_android_cb)
!#safari_cb_affinity(general)
@@||fresherslive.com^$generichide
@@||universoformulas.com^$generichide
@@||sneakernews.com^$generichide
@@||sonyliv.com^$generichide
@@||synology-forum.de^$generichide
@@||news.goo.ne.jp^$generichide
@@||agazeta.com.br^$generichide
!#safari_cb_affinity
!#endif
```

**Review by zloyden — CHANGES_REQUESTED** [review]

File: AnnoyancesFilter/sections/antiadblock.txt (hunk 1 above)
Comment (wout-codes) [reply AnnoyancesFilter/sections/antiadblock.txt:391]: It works for me

Comment (wout-codes) [reply AnnoyancesFilter/sections/antiadblock.txt:391]: What works for you I will add it.

Comment (wout-codes) [reply AnnoyancesFilter/sections/antiadblock.txt:391]: image
After killing the dialog there is another notice at the below of the page just like `geekzone.co.nz` issue.

Comment (zloyden) [reply AnnoyancesFilter/sections/antiadblock.txt:391]: Those CSS rules are enough for the website.

Comment (wout-codes) [reply AnnoyancesFilter/sections/antiadblock.txt:391]: @zloyden So should I remove the Scriptlet rule?

Comment (zloyden) [reply AnnoyancesFilter/sections/antiadblock.txt:391]: Yes, but not the scriptlet rule, ofc, just the domain)

**Review by zloyden — APPROVED** [review]

File: AnnoyancesFilter/sections/antiadblock.txt (hunk 1 above)
Comment (zloyden) [reply AnnoyancesFilter/sections/antiadblock.txt]: Well done:)

**Pull request merged**